lib/widgets/recent_dm_conversations.dart

```diff
@@ -140,7 +145,19 @@ class RecentDmConversationsItem extends StatelessWidget {
                 ),
                 maxLines: 2,
                 overflow: TextOverflow.ellipsis,
-                title))),
+                TextSpan(children: List.generate(users.length, (index) {
+                  final user = users[index];
+                  return TextSpan(text: user.name, children: [
+                    if (user.isBot) ...[
+                      const WidgetSpan(child: SizedBox(width: 5)),
+                      const WidgetSpan(
+                        child: Icon(ZulipIcons.bot, size: 15,
+                          color: Color.fromARGB(255, 159, 173, 173)),
+                        alignment: PlaceholderAlignment.middle),
+                    ],
+                    if (index < users.length - 1) const TextSpan(text: ', '),
+                  ]);
+                }).toList())))),
```

Comment:
I see a comment on the `switch`:

```dart
 // TODO dedupe with DM items in [InboxPage]
```

and that reminds me that we also show users' names in the Inbox page, when there are unread DMs. Would you add a prep commit that makes a helper function that can be used in this page and the Inbox page too? Perhaps it can return a `TextSpan`, and it can live in either file or perhaps a new file.